src/compiler/language-service.ts

```diff
@@ -172,8 +170,7 @@ export const initializeLanguageServiceInstance = (
                  * test file for 1st time run after clearing cache because
                  */
                 return (
-                  entry[1].modulePaths.find(modulePath => modulePath === fileName) &&
-                  !hasOwn.call(memoryCache.files, entry[0])
+                  entry[1].modulePaths.find(modulePath => modulePath === fileName) && !memoryCache.files.has(entry[0])
```

Comment:
I modified our internal memory cache in #1574 but forgot this one. It doesn't cause any bugs or performance issue but '_the fastest code is the code which does nothing_' so it's better to correct this condition.